### PR TITLE
OS X compatible buildtools.sh

### PR DESCRIPTION
**Pull request by jvdoorn**
OS X doesn't have `wget` installed by default, so before attempting to use `wget` check if the user is running OS X (Darwin) and if so use `curl` (shipped with OS X).

### CODE REVIEW

File: scripts/buildtools.sh [hunk 1]
```diff
@@ -53,5 +57,3 @@ if [ $is_183 -ne 0 ]; then
 else
     echo "CraftBukkit 1.8.3 installed; skipping BuildTools..."
 fi
-
-popd
```
Comment (mdcfe) [inline scripts/buildtools.sh]: Why did you remove this line?

The `popd` is necessary to make sure the build script is in the right working directory after running BuildTools.

**Review by mdcfe — CHANGES_REQUESTED** [review]

File: scripts/buildtools.sh (hunk 1 above)
Comment (jvdoorn) [reply scripts/buildtools.sh]: Whoops, looks like I removed that on accident, let me fix it

**Review by mdcfe — APPROVED** [review]
